MCprep_addon/conf.py

```diff
@@ -177,16 +300,12 @@ def icons_init():
 		for iconset in collection_sets:
 			preview_collections[iconset] = ""
 
-
+# ! Deprecated as of MCprep 3.4.2
 def log(statement, vv_only=False):
-	"""General purpose simple logging function."""
-	global v
-	global vv
-	if v and vv_only and vv:
-		print(statement)
-	elif v:
-		print(statement)
-
+    if env.verbose and vv_only and env.very_verbose:
+        print(statement)
+    elif env.verbose:
+        print(statement)
```

Comment:
Per comment somewhere lower, maybe we should just have a second `def debug` function, instead of a log with a vv_only flag.